include/rocksdb/options.h

```diff
@@ -1600,6 +1600,10 @@ struct DBOptions {
   // persisted replication sequence, so the max limit here shouldn't be quite
   // large
   uint32_t max_num_replication_epochs = 100;
+
+  // If true, further manifest writes will be skipped on the first manifest
+  // write error. Db has to be reopened to clear the error status
+  bool skip_manifest_write_on_first_manifest_write_error = false;
```

Comment:
We are not skipping manifest write, we are erroring it.

A better name might be `bool attempt_recovery_after_manifest_write_error`. Make it default to true, which is the existing behavior.